test/MathUtils.test.cpp

```diff
@@ -222,11 +222,16 @@ TEST(MathUtils, isRectInRectClass)
 TEST(MathUtils, mapDomainToRange)
 {
 	// Fahrenheit to Celcius
+	EXPECT_NEAR(-18.33333f, NAS2D::convertDomainToRange(-1.0f, 32.0f, 212.0f, 0.0f, 100.0f), 0.0001f);
 	EXPECT_NEAR(0.0f, NAS2D::convertDomainToRange(32.0f, 32.0f, 212.0f, 0.0f, 100.0f), 0.0001f);
+	EXPECT_NEAR(-17.77777f, NAS2D::convertDomainToRange(0.0f, 32.0f, 212.0f, 0.0f, 100.0f), 0.0001f);
+	EXPECT_NEAR(-17.22222f, NAS2D::convertDomainToRange(1.0f, 32.0f, 212.0f, 0.0f, 100.0f), 0.0001f);
```

Comment:
The "around zero" argument would make more sense for this method for an `unsigned` data type. For `signed` data types like these temperature conversions, particularly from degrees F, there doesn't seem to be anything particularly special about 0.

If it's a well know conversion point, that's fine. In particular, 0 degrees C is the melting/freezing point of water. If not, and I'm not aware of anything special happening at 0F, other than it's a nice round number and maybe people have memorized this, it maybe starts to approach the realm of [Cargo Cult Programming](https://en.wikipedia.org/wiki/Cargo_cult_programming).

With that said, we might perhaps still want to add testing where `InputType` is an `unsigned` quantity, and include a zero point there.